### ISSUE TITLE
Bad navigation on docs page for ActionBar

### ISSUE THREAD

**Issue by LukeHowellDev** [NativeScript/docs#117]
I cannot find code related to this in the docs repo but the "Next" for Action Bar should navigate to Dialogs (https://docs.nativescript.org/ui/dialogs) instead of the Action Item anchor (https://docs.nativescript.org/ui/action-bar#action-item)

<img width="1032" alt="Screenshot 2023-10-25 at 3 11 11 PM" src="https://github.com/NativeScript/docs/assets/202615/e3a05889-8cd1-4ac4-b00f-6fc22ca63e33">

**Comment by LukeHowellDev** [comment]
Thi s appears to be the same for Dialogs as well

**Comment by rigor789** [comment]
Good catch - I'll fix this when I get the chance, there are hidden sidebar items that drive these links, have to update the logic to ignore hidden items (these are used for various other things, just not show in the sidebar itself).
